index.js

```diff
@@ -261,7 +261,7 @@ client.on('messageCreate', async (message) => {
     process.exit();
 
   if (cmd !== 'eval') {
-    if (!cmd.match(/^[\w-]+$/) || !fs.existsSync(`./commands/${cmd}.js`))
+    if (!cmd.match(/^[\w-\$_]+$/) || !fs.existsSync(`./commands/${cmd}.js`))
```

Comment:
```suggestion
    if (!cmd.match(/^[\w-\$]+$/) || !fs.existsSync(`./commands/${cmd}.js`))
```

`\w` is equivalent to `[a-zA-Z0-9_]` and as such the additional `_` is redundant.

Reply:
wait I'm high aren't I

Reply:
nvm me thinking I'm high was the high part

Reply:
managed to to misread that as a capital for a moment